src/commands/scan/create.ts

```diff
@@ -164,24 +167,46 @@ async function setupCommand(
       }
     )
 
+  // TODO: I think the cwd should be set to the DIR|FILE arg of this command and the DIR/FILE be either '.' or the filename() of the arg
+  const cwd =
+    cli.flags['cwd'] && cli.flags['cwd'] !== 'process.cwd()'
+      ? String(cli.flags['cwd'])
+      : process.cwd()
+
+  const [orgSlug, ...targets] = cli.input
+
   const packagePaths = await getPackageFilesFullScans(
     cwd,
-    cli.input,
```

Comment:
This would have sent the org as part of potential folders (which probably would lead to an error? or accidental include maybe?). Not sure how blocking this was but it's fixed now.